### ISSUE TITLE
[RFC] GT++ new multi recipes with new tiers

### ISSUE THREAD

**Issue by 0lafe** [GTNewHorizons/NewHorizons#3730]
#### Which modpack version are you using?
2.0.5.1

#### What do you suggest instead/what changes do you propose?

Based on what was in #3701 I made some new recipes to make the multis harder to make, as well as put them in more fitting tiers. 
![all of ev](https://user-images.githubusercontent.com/21280852/46711651-447c9a00-cc1b-11e8-949d-a8067e77b31d.PNG)
![iv to luv](https://user-images.githubusercontent.com/21280852/46711660-46def400-cc1b-11e8-91ec-0f7de3aa00fc.PNG)
![zpm to uv](https://user-images.githubusercontent.com/21280852/46711661-49414e00-cc1b-11e8-81f4-d74b3d9e114b.PNG)
![uhv](https://user-images.githubusercontent.com/21280852/46711664-4b0b1180-cc1b-11e8-9d57-354ac6277c7c.PNG)

Missing recipes right now are: mixer, sifter and the cutting machine 

I put them in a spreadsheet too if anyone wants to just look at that instead. It shows what the current recipe is, and a proposed one. Totally willing to change it as I get feedback, but these are some starting recipes I came up with. 

General ideas: better tiering, more expensive compared to singleblocks, more tier fitting as well

**Comment by Highbeam** [comment]
I thought the original idea was to make the recipe harder but keep it within the HV-ZPM tiers (the tiers that most people realistically play)..  Here you've moved everything to ZPM and beyond again...  I see UV, UHV, UEV in there so I'm not behind this sorry.

I like some of the recipe ideas but I don't see why all these multis need to be moved this high....

1 note: if the Packager has a complex recipe for the Supply Depot Casings then I believe the Supply Depot Casings shouldn't be in the Controller recipe.

**Comment by Ghengis88** [comment]
I agree with Highbeam, the tiers should not be changed, just make them more expensive

**Comment by 0lafe** [comment]
>UV, UHV, UEV

Those are circuits you're seeing. Check the assembler recipes for the casings to see the tier they're in. There's 3 past zpm, all in UV, and all deserving of being in UV. Got rid of the UHV on assembler too because no one wanted it at UHV. For the most part each recipe uses the highest tier circuit possible, requiring you to have the circuit assembler of that tier. I think that pushes the EV stuff to past T2 rocket, and most things get unlocked Midway through a tier. 

>I agree with Highbeam, the tiers should not be changed, just make them more expensive

Except they can't really be both for more than a tier or two. There's not much in EV or IV that would still be considered expensive in LuV+ especially when that tier pretty much gives you all the ores you could ever want. These multis are good, damn good, and imo need to be tiered to fit that. No reason imo you need a super fast wiremill at IV, or a bender, or much else. For the most part the gt machines can carry you all the way, and the GT++ ones are just an extra help, but if that's all they are, they should be gated imo. They're just a straight buff to single blocks, so while more cost is a big part of it, no one really needs that power pre LuV. Some of the stuff (ebf, vac freezer, assembler) is really, really good. Better than pretty much anything else you can make in any of the later tiers. You can already usually make way more power than you use, so I don't get why we need to let people use even less power for faster speeds. I get that people want them "because PAs suck" which I agree with kinda. They have their use, but they can't really replace a single, single block machine (unless you just wanna put it in the PA for better item management). However the GT++ multis will always be significantly better than any single block, with speeds usually faster than the best possible array, especially around EV/IV. It's huge amounts of speed in a tier that no one needs huge amounts of speed

@draknyte1
@cinobi 
@Teirdalin
@richardhendricks
@DoomSquirter

You probably have an opinion or two

**Comment by Teirdalin** [comment]
I know it may be easy to make UV, ZPM, and UHV circuits and the like for us who have no lives;  but increasing the recipes this much means that the majority of people won't even bother with the fun multis.

**Comment by Ghengis88** [comment]
I had a discussion with Bryfer erlier today about this, we arrived at the conclution that the multiblocks should be as they are now but that the main problem is that they are to easy to upgrade to the next tier.

as a solution to this Bryfer had the idea that for a machine to work it has to have the singelblock conterpart in a slot in the controller. then you would need to craft all the singelblock machines but have the benefit of the multiblocks

so fo example to have the macerator work in LuV you need to place a Elite Pulverizer in the control block inventory and change the energy hatch.

**Comment by 0lafe** [comment]
ZPM circuits can be made in IV though. IV is where most of the multis are already. UV circuits are LuV, which is also where some multis are right now. UHV is zpm, which has stuff gated past it. People who don't see these multis won't see mk3 fusion either, or an assembly line. If they don't see them, they don't see them. I put them where I put them because of how good they are, not how fun. Also, they aren't really interesting. They're single blocks that look different, and run for less EU/t + faster. People can play how long they wanna play, but I never understood that as an argument for making things available earlier. 

Each one *should* be cheaper than making a PA I think, while costing higher tier stuff, to put it at a more appropriate tier.

**Comment by 0lafe** [comment]
>I had a discussion with Bryfer erlier today about this, we arrived at the conclution that the multiblocks should be as they are now but that the main problem is that they are to easy to upgrade to the next tier.

as a solution to this Bryfer had the idea that for a machine to work it has to have the singelblock conterpart in a slot in the controller. then you would need to craft all the singelblock machines but have the benefit of the multiblocks

so fo example to have the macerator work in LuV you need to place a Elite Pulverizer in the control block inventory and change the energy hatch.

That works for some, but I really don't like it because that's super cheap to upgrade. If the multis are in EV/IV by LuV+ the expensive part becomes the single block, which means pretty much a huge speed bonus/EU discount is applied to every single block that has a GT++ multi, and is after IV. Imo that's dumb. It shows how the GT++ multis are OP though, when this is an increased cost and is pretty damn OP.

**Comment by Ghengis88** [comment]
Make it a singel block machine and a special GT++ circuit that is expensive tier apropriate then. But i really think pushing the machines to later tiers is the worst possible idea. I think the GT++ placed the machines at very good tiers to motivate players and make them enyoy the game, but at this this discussion says they are to cheap at the moment, but not at the time you first can build them they are not

**Comment by Highbeam** [comment]
I refer to my previous suggestion:  "The Uber Hatch".   An Uber Hatch for each tier will essentially determine that multi's maximum tier.  "Uber Hatch IV" in the multiblock structure to limit/upgrade the maximum tier recipe of the multi to IV...  At least this would be needed for the Assembler...

You have to think about the playthrough and how it goes through so many stages.  By EV-ZPM you really are thrashing out your base design and changing it based on new multiblocks/single machines and yes this is the 'fun' part.  These multi's really inspired and changed my thought process on how I wanted to design my base.  If I did it on single block machines or PA it would've been different.

Since all these machines/multis really make the 'core' machines and are essentially upgrades for what you're already using I fail to see any reason to push them up to high tiers.  It takes several months to get to ZPM as things stand, especially with all the recent changes and making the game harder all the time.  

I agree with @Ghengis88 that these multis aren't a problem where they are right now.  Just the recipe can be made a bit harder and the problematic recipes addressed.

It's probably easier to change the recipe to require something thats tier dependent

**Comment by 0lafe** [comment]
>Make it a singel block machine and a special GT++ circuit that is expensive tier apropriate then. But i really think pushing the machines to later tiers is the worst possible idea. I think the GT++ placed the machines at very good tiers to motivate players and make them enyoy the game, but at this this discussion says they are to cheap at the moment, but not at the time you first can build them they are not

While that helps with part of it, a big part is I don't think those tiers should be getting speeds like this. GT++ did a terrible job of fitting them into NH, they have no place a little more than halfway through, they're really, really good and should be put in a tier based on that. It's like you get a zpm wiremill in IV, and instead of putting it back in ZPM we're discussing what EV parts it should take. It's still a cost associated with it, however the cost at EV to make an LuV multi is to make it to LuV then make the multi. That huge cost isn't really emulatable with a single block + circuit imo, nor would I really want to try.

**Comment by 0lafe** [comment]
>Since all these machines/multis really make the 'core' machines and are essentially upgrades for what you're already using I fail to see any reason to push them up to high tiers. It takes several months to get to ZPM as things stand, especially with all the recent changes and making the game harder all the time.

I think you're missing my point. I don't agree that the speed bonuses/EU discounts associated with GT++ machines should be available as early as they are. It's less about tiering recipes, but simply their speed is way too good for the cost. They do something no other machine really can do, they speed up GT machines. That's a really good and really useful feature. 

I'm not balancing them for time played, or aesthetics, but for what they bring to the game. An overpowered speed increase combined with an overpowered power decrease. I don't see any way that these fit in such early tiers. You should have to play a while to get them. If people don't get to some, then they don't get to some. It should take that long to get this kind of stuff, it's a long modpack, it's a long game, I really fail to see the problem with putting them farther into the game. I played for about 2 months and I'm close to being able to make most of the multis already (EV, T1 rocket). They're really early for what they do, no one needs the bonuses they give. They should take a while to get, they're busted

**Comment by Dream-Master** [comment]
@0lafe @Highbeam @Ghengis88 @Teirdalin did @draknyte1 allready know about this recipe wishes?

**Comment by Ghengis88** [comment]
he saw the last post atleast @Dream-Master

**Comment by Dream-Master** [comment]
ok

**Comment by draknyte1** [comment]
@0lafe they're not moving beyond the tiers I've already specified, so give up with your UV shit.

**Comment by 0lafe** [comment]
>@0lafe they're not moving beyond the tiers I've already specified, so give up with your UV shit.

So there's 3 in UV, does that mean that the rest are A OK and I can put the rest in ZPM? Also, I can maybe get the assembler at ZPM, I mean I never even made a UV assembler because it was kinda shit. But the ebf is pretty OP, especially at ZPM doing half time neutronium/naq alloy. No UV+ ebf recipes though which is why I thought UV was better for it. Vac freezer I put with it just because they're similar but I guess it could go a little earlier to zpm with the assembler 

@draknyte1

**Comment by Scottishprog** [comment]
Olafe,

I see your point....

But at what point does "proper" tiering conflict with playability?  Even for a group as dedicated as this one?
To touch on a point that @Highbeam  stated, you are at some point semi-finalizing the base design. So the machines need to be available at some point, so they can be slotted into the design! (There is no CAD package to act as a planning substitute...)

I think it comes down to a value judgement.  And I am on the side that the availability is needed about where it is. Just so things don't bog down. (Note I didn't say that the price needed to stay the same...)

Unfortunately, that doesn't deal with the issue of not so good tiering at the higher levels...



To put it another way, I believe if the Multiblocks were tiered up, the recipes they used would need to be looked at as to not spike the demand for power too much....

Also,  I have been playing my current world for rather more than two months, and still at LV. I would hesitate to _not_ include the experiences of the slower players in balancing.  A tier that is never reached....

**Comment by 0lafe** [comment]
If you're in LV for 2 months then that's your choice/you don't play much/you're doing something really really wrong lol. I think it should be balanced around rushing through the game, and going for the most efficient options. I'm doing the same as you, I'm not sure when I'll see the end of EV, and I've been spending like 2 weeks making a golem farm for golems. It's been fun, but I'm not sitting here complaining that I can't get past EV because I know I can, I know what I'd need to do, I'm just not doing it yet (that kinda sounded like I meant it towards you, totally didn't mean that, just my opinion). 

>I think it comes down to a value judgement. And I am on the side that the availability is needed about where it is. Just so things don't bog down. (Note I didn't say that the price needed to stay the same...)

My biggest reason for this is that I've gone through the tiers. All the way to UMV without a GT++ multi because there aren't needed (oh and they totally weren't working until the version after I lost my world). I feel confident saying that you really don't need them ever. I never made a PA until LuV and only used them for ore processing. I had 1-16 HV of most of the crafting ones, and that speed was what I used until I finished an MK3 fusion setup making about 4M EU/t and felt like I should spend some more power. If I used an underpowered setup and had no issues, I think the progression is fine without gt++. Considering I could have upgraded most of my processing much earlier, just didn't because the speed wasn't needed. 

The only stuff that's really changed is the moving of IV solars to LuV (kinda rolled back to IV though considering the GT++ assembler multi works now), oil is different, and circuits. Unless circuits alone take like 16x faster processing than anything else put together, then I really don't see what people need all this speed for. Other than the assembly line there's not much that takes materials in an impactful way until fusion, which is 4 things across 4 tiers. The same amount of tiers that take you from basic electricity to Mars and nukes. 

>Also, I have been playing my current world for rather more than two months, and still at LV. I would hesitate to not include the experiences of the slower players in balancing. A tier that is never reached....

So kinda, but I feel like the most useful insight usually comes from people affected by whatever the issue is. People with personal experience I guess I mean. If you're still in LV I don't think the problem is GT++ multis.

**Comment by mitchej123** [comment]
>If you're in LV for 2 months then that's your choice/you don't play much/you're doing something really really wrong lol
>  I think it should be balanced around rushing through the game, and going for the most efficient options. 

I think it should be balanced somewhere in the middle... not for either extreme

**Comment by 0lafe** [comment]
I guess I kinda phrased it wrong. I think that it should always be preventative in letting people rush all the way through everything. However it shouldn't be required. However I think current IV+ is pretty lax in the requirements of infrastructure. If you get 4x power per tier kinda, I feel like you should be more required to use it.

**Comment by Scottishprog** [comment]
@0lafe 

To be fair... I am very late LV, and I have not been putting in max time.

But that's my point... not everyone is moves up the tiers quickly.

**Comment by Highbeam** [comment]
0lafe I get that you keep saying you've played the pack fully etc etc.  What I still cannot agree on is this 'gt++ aren't necessary' attitude.  At the end of the day, if the multi's weren't there - I would have a hundred machines sat somewhere processing all the different stuff that I currently use the multis for.  They shave off time, server resources, space in base and look a lot more interesting than having rows of single blocks or PA's...

Even after I have them, my progress hasn't suddenly become rapid!  But the things I do have been less tedious,  waiting around for stuff to craft for hours or using a hundred machines to craft the same thing.  Still agree with expensive recipes for the multiblocks but I say yet again, review the recipes that are the problem...  If you put in materials that cannot be obtained until that tier, they will be fixed.

**Comment by DoomSquirter** [comment]
I can't state how perfect highbeam's statement is.  it hits the nail square on the head.  with gt++ you consolidate and not advance.  it doesn't give you an advantage, moreso it gives you some consolidation of existing tools/processes w/o any advantage that one might consider bypassing other things (aka OP).

**Comment by Ghengis88** [comment]
I think that Highbeam's Idea about the Uber hatch is even better then what Bryfer and I came up with, it would solve all the things i finde problematic with the GT++ Multiblocks

**Comment by Highbeam** [comment]
Ok just edited it again:

In a nutshell:

LuV+ = Energy Hatch requires same tier superconductor when crafting.
LuV Energy Hatch needs LuV superconductor
ZPM Energy Hatch needs ZPM superconductor etc

Optional:  pre LuV Energy Hatch requires same tier Pump.
MV Energy Hatch needs MV Pump
HV Energy Hatch needs HV Pump etc

GT++ Multiblocks structure ONLY allows 1 energy hatch to prevent running higher tiers.

**Comment by richardhendricks** [comment]
I like this idea.  I think the pumps shouldn't be required until HV, and along with the pump we should include the appropriate large cell for that tier filled with ic2 coolant.

**Comment by DoomSquirter** [comment]
I really don't get the hangup on speed.  I really don't.   it's not like a single box that gives you 12x ore multiplication or reduces power usage by 50% or whatever.  I don't think we need an accountant to go in and itemize every side of this gt++ multiblock argument and ensure it's perfectly balanced as long as the player isn't getting some form of advantage from it.

**Comment by 0lafe** [comment]
>I really don't get the hangup on speed. I really don't. it's not like a single box that gives you 12x ore multiplication or reduces power usage by 50% or whatever. I don't think we need an accountant to go in and itemize every side of this gt++ multiblock argument and ensure it's perfectly balanced as long as the player isn't getting some form of advantage from it.

It doesn't really matter if it's a single or a multi, I don't think speed like that should be available in those tiers. It's a pretty decent advantage I'm not sure what you mean, it's faster items processed and less EU used. Kind of the only 2 things that a machine does. Takes a time and a power, and these take noticable less of both, which is an impactful advantage imo. If speed doesn't matter that much I'd suggest removing the speed/EU bonuses and increasing the processes/tier to 4. With the current speed/EU discount, I don't see them fitting into EV/IV.

**Comment by Teirdalin** [comment]
Wait-  it reduces energy cost to less than what the processing of the item would normally take?

**Comment by 0lafe** [comment]
If you can get alk to agree that's ideal, but I've been barking up that tree since 2.0.3. It's a %speed increase. Centrifuge has 2.25x speed bonus + 90% EU consumed. So infinity comb processing would take 2.25x less time and use 90% the EU that even that would take. Yes it's OP lol. Specifically put the centrifuge past other ore processing because of the bee bonus with it.

**Comment by Teirdalin** [comment]
Maybe the machines could have an upgraded form which gives the power discount and a normal form which is full eu cost per process with just the speed boost since-  entire point of a multi to begin with aside from customization; while later game you'd be doing whole stacks of processing with arrays rather than singles anymore;  so an upgraded form which gives a discount could be useful for making GT++ multis good later game while still having them be nice mid game.

**Comment by Scottishprog** [comment]
That's a possibility....

Full speed gain now, and put in higher level single block (+1 tier) to get the power discount.

I agree with that approach.

**Comment by 0lafe** [comment]
>Maybe the machines could have an upgraded controller form which gives the power discount and a normal form which is full eu cost per process with just the speed boost since- entire point of a multi to begin with aside from customization.
While later game you'd be doing whole stacks of processing with arrays rather than singles anymore; so an upgraded form which gives a discount could be useful for making GT++ multis good later game while still having them be nice mid game. Although if it's only 2.25x speed over a single block, that's not necessarily that exciting of a machine really; maybe if it could have its speed upgraded with perhaps height or even a new processing casing or something which increase amp cost to keep it running but gives an additional 1x or 1.15x speed increase increment; that'd be pretty cool.

I like where you're going with amps. Something where the hatch could draw multiple amps and for each one it would add a 100% speed bonus. So like 4A and it goes 4x faster + parallel stuff if necessary. I love the idea, but now that I think about it, it sounds more OP than the current stuff and I'd rather see it even later tiered which I don't think would go well. 

Problem is alk. He won't really do much in the way of "change mechanics for gtnh". If you can convince him great, but I've only seen him oppose it. 

Now uh, I don't her why you mind the 90% EU part more than the 2.25x part. 2.25x speed means 2.25x less EU, which is a lot more than 90%. Im a little concerned about the larger ones (60-50%) and if they let more overclocks happen because 40000eu/t would be solid LuV instead of zpm. Stuff like that could be an issue too, but I haven't looked at it enough to know if/how it'll inpact anything. 

>Full speed gain now, and put in higher level single block (+1 tier) to get the power discount

I'd rather see it the other way around as the speed is clearly the more OP of the two. It's an EU discount usually more than the % discount, and runs faster

**Comment by draknyte1** [comment]
> Problem is alk. He won't really do much in the way of "change mechanics for gtnh". If you can convince him great, but I've only seen him oppose it.

I’m not adding new mechanics or new blocks. With good reasons for each... by now I’ve elaborated on this enough times for you to realise it won’t ever happen. What do you expect for free?

**Comment by 0lafe** [comment]
I'm not complaining about it, just saying we need to stick to stuff like recipe changes

**Comment by mitchej123** [comment]
>So infinity comb processing would take 2.25x less time and use 90% the EU that even that would take. Yes it's OP lol. Specifically put the centrifuge past other ore processing because of the bee bonus with it.

Uh, saying it's OP because of late game mats, and should be nerfed for everyone/early game, is a bit silly.

**Comment by Teirdalin** [comment]
> > So infinity comb processing would take 2.25x less time and use 90% the EU that even that would take. Yes it's OP lol. Specifically put the centrifuge past other ore processing because of the bee bonus with it.
> 
> Uh, saying it's OP because of late game mats, and should be nerfed for everyone/early game, is a bit silly.

That's true;  I suppose it's minor enough not to really be an issue really.

**Comment by DoomSquirter** [comment]
y'all just need hugs.  that's all.  lots of hugs.  maybe some puppies or baby beavers but hugs first, cuz someone hurt you at some point and I feel sorry for ya.... 😢

**Comment by Dream-Master** [comment]
@draknyte1 are you accepting pulls when we need to change some machine recipes? adding them with Minetweaker is not the best way. GTNH need sometimes different balancing then the rest of Vanilla GT.
